src/wallet/wallet.cpp

```diff
@@ -3376,7 +3376,6 @@ bool CWallet::IsTxAssumed(const CWalletTx& wtx) const EXCLUSIVE_LOCKS_REQUIRED(c
     return false;
 }
 
-
```

Comment:
On the`gui:` comit fff5cb42dfe91ce1ba1a1aa2269253f262c6e2b4, was this unintentional?

Reply:
Dropped